manual/softlayer-sru-20.2.45.txt

```diff
@@ -0,0 +1,1161 @@
++++ AZURE_BOOT_DIAG=storeitsru
++++ AZURE_ADVANCED_NIC_TESTS=true
++++ GCE_ZONE=UNSET
++++ OPENSTACK_ADMIN_NET_ID=UNSET
++++ OPENSTACK_SSH_KEY_NAME=UNSET
++++ SSHOPTS=(-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o LogLevel=ERROR)
+++ SSHOPTS=(-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o LogLevel=ERROR)
++ parse_args ./sru-scripts/manual/softlayer-sru xenial
++ script=./sru-scripts/manual/softlayer-sru
++ shift
++ '[' 1 = 0 ']'
++ '[' 1 -ne 0 ']'
++ case "$1" in
++ UBUNTU_RELEASE=xenial
++ shift
++ '[' 0 -ne 0 ']'
++ '[' -z xenial ']'
++ assert_expected_vars PROPOSED_SCRIPT LP_USER SSHOPTS
++ local value=
++ for var in $@
++ eval 'value=$PROPOSED_SCRIPT > /dev/null'
+++ value=setup_proposed.sh
++ '[' -z setup_proposed.sh -o setup_proposed.sh = UNSET ']'
++ for var in $@
++ eval 'value=$LP_USER > /dev/null'
+++ value=chad.smith
++ '[' -z chad.smith -o chad.smith = UNSET ']'
++ for var in $@
++ eval 'value=$SSHOPTS > /dev/null'
+++ value=-o
++ '[' -z -o -o -o = UNSET ']'
++ create_samplecloudconfig SAMPLE_CLOUDCFG
++ filename=SAMPLE_CLOUDCFG
++ cat
++ create_setup_proposed_script
++ '[' 0 -eq 0 ']'
++ cat
++ echo '### BEGIN xenial'
+### BEGIN xenial
+++ launch-softlayer --image=os:xenial -u sethostname.yaml
+++ awk '/primary ip/{printf "root@%s", $NF}'
+^[[B^[[A [is_ready=True status=no-status]
+Destroy instance with:
+  slcli virtual cancel 104583766
++ VM_IP=root@75.126.139.189
++ sleep 100
++ retry_ssh_command root@75.126.139.189 'cloud-init status --wait --long' 5
++ local vm_ip=root@75.126.139.189 'cmd=cloud-init status --wait --long' sleep_seconds=5
++ ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o LogLevel=ERROR root@75.126.139.189 -- cloud-init status --wait --long
+
+status: done
+time: Tue, 23 Jun 2020 05:06:42 +0000
+detail:
+DataSourceIBMCloud [OS-Code/Live /dev/xvdh]
++ ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o LogLevel=ERROR root@75.126.139.189 -- dpkg-query --show cloud-init
+cloud-init	19.4-33-gbb4131a2-0ubuntu1~16.04.1
++ ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o LogLevel=ERROR root@75.126.139.189 -- cat /run/cloud-init/result.json
+{
+ "v1": {
+  "datasource": "DataSourceIBMCloud [OS-Code/Live /dev/xvdh]",
+  "errors": []
+ }
+}
++ ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o LogLevel=ERROR root@75.126.139.189 -- '!' grep Trace /var/log/cloud-init.log
++ retry_ssh_command root@75.126.139.189 systemd-analyze 5
++ local vm_ip=root@75.126.139.189 cmd=systemd-analyze sleep_seconds=5
++ ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o LogLevel=ERROR root@75.126.139.189 -- systemd-analyze
+Startup finished in 6.608s (kernel) + 3min 6.909s (userspace) = 3min 13.517s
++ ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o LogLevel=ERROR root@75.126.139.189 -- systemd-analyze
+Startup finished in 6.608s (kernel) + 3min 6.909s (userspace) = 3min 13.517s
```

Comment:
Is this expected for xenial, a boot time of 3 min ? I know that this is happening on both current and proposed versions. but I still think it is odd

Reply:
Yeah crazy right. softlayer on xenial I believe has always taken 3 mins to boot. I'm not certain what their backplane provisioning issue is here, but I don't think we've dug into it on this platform.